Sources/Kevin/Core/Kevin.swift

```diff
@@ -24,7 +24,14 @@ final public class Kevin {
 
     public static let shared = Kevin()
     
-    private init() { }
+    private init() {
+        if let preferredLanguage = Locale.preferredLanguages.first {
```

Comment:
Small nit pick, maybe use guard statement here?

Reply:
I was considering guard statement, but it requires a `return` statement in `else` clause and I am not a fan of having early `return`s in init function, because it can get confusing down the line